### PR TITLE
add --cluster to the client

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/308)

### CODE REVIEW

**Comment by yagonobre** [comment]
@aguerra only in the code or the flag name too?

**Comment by aguerra** [comment]
@yagonobre Both.

**Comment by yagonobre** [comment]
@aguerra sounds good

File: pkg/client/config.go [hunk 1]
```diff
@@ -74,11 +74,14 @@ func SaveConfigFile(path string, cfg *Config) error {
 	return ioutil.WriteFile(path, b, 0600)
 }
 
-func GetConfig(cfgFile string) (*ClusterConfig, error) {
+func GetConfig(cfgFile, cfgContext string) (*ClusterConfig, error) {
 	cfg, err := ReadConfigFile(cfgFile)
 	if err != nil {
 		return nil, err
 	}
+	if len(cfgContext) > 0 {
```
Comment (drgarcia1986) [inline pkg/client/config.go]: maybe `if cfgContext != ""` sounds more simple (avoid unecessary process).

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Next Release] - Release Date
 ### Added
 - [server] Add --debug flag. For now only print the stack trace on panic/recover.
+- [client] Add --context flag. To use a cluster different of current-cluster.
```
Comment (drgarcia1986) [inline CHANGELOG.md]: Isn't a problem in your PR but this entry already in `Added` section of the changelog, for this reason, that word `add` is unnecessary.

**Comment by yagonobre** [comment]
@drgarcia1986, what you thin about the flag name?

**Comment by drgarcia1986** [comment]
I prefer `cluster` instead of `context` too.

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 updated

**Comment by yagonobre** [comment]
close 293

**Pull request merged**